### PR TITLE
feat: stableswap add limit to remove liquidity [A6]

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

File: pallets/stableswap/src/lib.rs [hunk 1]
```diff
@@ -179,15 +179,14 @@ pub mod pallet {
...
+			assets: Vec<AssetBalance<T::AssetId>>,
 		},
 		/// Liquidity removed.
 		LiquidityRemoved {
 			pool_id: T::AssetId,
 			who: T::AccountId,
 			shares: Balance,
-			asset: T::AssetId,
-			amount: Balance,
+			amounts: Vec<AssetBalance<T::AssetId>>,
```
Comment (Roznovjak) [inline pallets/stableswap/src/lib.rs:189]: This change is not necessary

File: pallets/stableswap/src/lib.rs [hunk 2]
```diff
@@ -565,15 +569,19 @@ pub mod pallet {
 			)
 			.ok_or(ArithmeticError::Overflow)?;
 
+			ensure!(amount >= min_amount_out, Error::<T>::MinimumAmountNotReached);
```
Comment (Roznovjak) [inline pallets/stableswap/src/lib.rs:572]: Is 0 a valid value for the amount? Can it be 0?

File: pallets/stableswap/src/lib.rs (hunk 1 above)
Comment (enthusiastmartin) [reply pallets/stableswap/src/lib.rs:189]: it is preparation for remove_liquidity where you obtain all assets. This functionality will be added later. 

so just to make sure that if anything/anyone already works with events, there wont be breaking changes.

File: pallets/stableswap/src/lib.rs (hunk 2 above)
Comment (enthusiastmartin) [reply pallets/stableswap/src/lib.rs:572]: you can specify 0 as min amount. up to you.

Comment (Roznovjak) [reply pallets/stableswap/src/lib.rs:572]: If a user can receive zero amount, then ok.

Comment (enthusiastmartin) [reply pallets/stableswap/src/lib.rs:572]: well, then we have some issue somewhere of he specify so very low amount that he does not get anything.

same thing is in omnipool and xyk.

**Pull request merged**